src/cpp/database/database.cpp

```diff
@@ -279,6 +280,25 @@ EntityId Database::insert_new_topic(
     return entity_id;
 }
 
+bool Database::is_type_in_database(
+        const std::string& type_name)
+{
+
+    return (type_idls_.find(type_name) != type_idls_.end());
```

Comment:
Remove extra line break
```suggestion
{
    return (type_idls_.find(type_name) != type_idls_.end());
```